IdentityCore/src/MSIDBrokerConstants.h

```diff
@@ -69,8 +69,10 @@ extern NSString * _Nonnull const MSID_BROKER_DEVICE_MODE_KEY;
 extern NSString * _Nonnull const MSID_BROKER_SSO_EXTENSION_MODE_KEY;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->